CRM/Contact/Page/AJAX.php

```diff
@@ -745,35 +745,35 @@ public static function getDedupes() {
     if (!empty($columnDetails)) {
       switch ($columnDetails['data']) {
         case 'src':
-          $orderByClause = " ORDER BY cc1.display_name {$dir}";
+          $orderByClause = " ORDER BY cc2.display_name {$dir}";
```

Comment:
No actually I think the change is correct - that is changing the behaviour when ordering by src - which is contact 2